lib/StoryEngine/DataLayer/model/GlobalRuleDataModel.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+
+namespace StoryEngine.StoryEngineDataModel;
+
+public enum HowManyDataModel { all, any }
+public enum ListTypeForNodesDataModel { tags, storyElements, nodeIDs }
+public enum NodeStateDataModel { seen, notSeen }
+public enum TagStateDataModel { present, notPresent }
```

Comment:
Similar question here about this being `isTaggedState` with true/false; apologies if these questions are my lack of C# knowledge showing 😅